sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/CatalogV2Util.scala

```diff
@@ -379,10 +379,6 @@ private[sql] object CatalogV2Util {
     }
   }
 
-  def loadRelation(catalog: CatalogPlugin, ident: Identifier): Option[NamedRelation] = {
```

Comment:
This is a test-only function. I found it as I was searching all the call sites of `loadTable`.